shopsafe-backend/src/main/java/com/google/sps/data/CountyStats.java

```diff
@@ -67,4 +78,46 @@ public double getCountyScore() {
         // TODO: Create a better scoring system for counties.
         return (percentageCounty - percentageUS) * 5000 + 5;
     }
+
+    /*
+     * Given a county, use Census API to find population
+     */
+    private static long getPopulation (County county) {
+        try {
+            String results = "";
+            String line;
+
+            // Read response of census api call
+            URL url = new URL(getAPIUrl(county.getCountyFips()));
+            BufferedReader reader = new BufferedReader(new InputStreamReader(url.openStream()));
+            while ((line = reader.readLine()) != null) {
+                    results += line;
+            }
+            reader.close();
+
+            //Convert to Json and parse for population
+            JSONArray jsonResult = new JSONArray(results).getJSONArray(1); //skip schema on first entry
+            return jsonResult.getLong(0);
+
+        } catch (Exception e) {
+            //Population not found
+            System.out.println(
+                "Unable to get population for " + county.getCountyName() + ", " + county.getStateName());
+            return 0;
```

Comment:
We are going to need to determine what we want to do when we get 0 for population.